inyoka_theme_default/templates/portal/usercp/settings.html

```diff
@@ -0,0 +1,31 @@
+{#
+    portal/usercp/settings.html
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    This page shows the user control panel settings page.
+
+    :copyright: (c) 2013-2015 by the Inyoka Team, see AUTHORS for more details.
+    :license: BSD, see LICENSE for more details.
+#}
+
+{% extends 'portal/usercp/overall.html' %}
+
+{% block breadcrumb %}
+  {{ super() }}
+
+  {{ macros.breadcrumb_item(_('Settings'), href('portal', 'usercp', 'settings'))}}
+{% endblock %}
+{% set selected = 'settings' %}
+
+{% block content %}
+  {% call macros.outer_form(csrf_token(), form, manually_rendered=True) %}
+    <h3>{% trans %}Settings{% endtrans %}</h3>
+      {{ macros.inner_form(form, ['notify', 'notifications','timezone']) }}
```

Comment:
just to be sure: 'ubuntu_version' is not rendered as it is ubuntuusers.de-specific?